pkg/core/tekton/constants.go

```diff
@@ -0,0 +1,21 @@
+package tekton
+
+const (
+	pipelineRunPrefix         = "fuseml-"
+	pipelineRunServiceAccount = "staging-triggers-admin"
+	workspaceAccessMode       = "ReadWriteOnce"
+	workspaceSize             = "2Gi"
```

Comment:
Just a note that we need to investigate how to dynamically adjust the size based on the contents (codesets, models, datasets, build results and so on) that it needs to accommodate.